bigbluebutton-html5/imports/api/users/server/methods/validateConnection.js

```diff
@@ -0,0 +1,65 @@
+import { Client } from 'pg';
+import upsertValidationState from '/imports/api/auth-token-validation/server/modifiers/upsertValidationState';
+import { ValidationStates } from '/imports/api/auth-token-validation';
+import userJoin from '/imports/api/users/server/handlers/userJoin';
+import Users from '/imports/api/users';
+
+import createDummyUser from '../modifiers/createDummyUser';
+import updateUserConnectionId from '../modifiers/updateUserConnectionId';
+
+async function validateConnection(requesterToken, meetingId, userId) {
+  try {
+    const client = new Client({
+      host: 'localhost',
+      port: 5432,
+      database: 'bbb_graphql',
+      user: 'bbb_frontend',
+      password: 'bbb_frontend',
+      query_timeout: 30000,
+    });
```

Comment:
This values should come from an env config.
Just like Redis `process.env.REDIS_HOST`.